### ISSUE TITLE
Tasks from template are not being created after ticket fails to open

### ISSUE THREAD

**Issue by felipebart10** [glpi-project/glpi#9828]
### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Version

9.5.5

### Bug description

When opening a ticket from a template that have a task to be created automatically, if we forget to fill a obligatory field and click on "Add" (thus failing to open and returning a warning to fill that field), the next time we try to open (with the obligatory field now filled), the ticket is opened, but the automatic tasks are not created. 

### Page URL

glpi.url/front/ticket.form.php

### Steps To reproduce

1. Start creating a new ticket
2. **Select a ticket category that have a template with a task to be created and have an obligatory field.**
3. Start filling all the ticket opening form and **purposely forget to fill an obligatory field**.
4. Click on "Add". The form will return a warning, asking you to fill the obligatory field left behind.
5. Fill the field and click on "Add" again. Ticket will be opened.
6. Go on "Processing ticket" tab (ticket timeline) and notice that configured tasks were not created.

### Your GLPI setup information

```markdown
GLPI 9.5.5 ( => /var/www/html/glpi)
Installation mode: TARBALL
```

### Anything else?

When this happens, our techs are trained to restart ticket opening process, which causes time waste. If ticket creation goes without error warnings, tasks will be created normally.

**Comment by felipebart10** [comment]
The issue still remains on 9.5.x

**Comment by felipebart10** [comment]
Issue still happening on GLPI 9.5.6

**Comment by cconard96** [comment]
Fixed via #10873.

**Comment by felipebart10** [comment]
Hello @cconard96! Thanks for doing it. Will this bug be fixed in 9.5.7 or only at GLPI 10?

**Comment by cconard96** [comment]
Fixed for version 10 since that is the next planned release.
